graph/typeDefs.graphql

```diff
@@ -626,7 +634,7 @@ type RootQuery {
   comment(id: ID!): Comment
 
   # All assets. Requires the `ADMIN` role.
-  assets: [Asset]
+  assets(query: AssetsQuery!): [Asset]
```

Comment:
As we currently allow querying all assets without any query parameters, I think `query` should be optional instead.